server/models/user.js

```diff
@@ -28,8 +52,36 @@ module.exports = function(sequelize, DataTypes) {
         var json = {
           id: this.get('id'),
           email: this.get('email'),
-          name: this.get('name')
+          name: this.get('name'),
```

Comment:
@dwoznicka what do you say if I ask you to do the same trick with user what we did with snippet ? I mean what if we have a `avg` field?